common/aws/dynamodb/client.go

```diff
@@ -156,6 +157,51 @@ func (c *Client) UpdateItem(ctx context.Context, tableName string, key Key, item
 	return resp.Attributes, err
 }
 
+func (c *Client) UpdateItemIncrement(ctx context.Context, tableName string, key Key, item Item) (Item, error) {
+	update := expression.UpdateBuilder{}
+	for itemKey, itemValue := range item {
+		if _, ok := key[itemKey]; ok {
+			// Cannot update the key
+			continue
+		}
```

Comment:
I realize this is using the template above, but it's unclear to me why this is failing silently.

Reply:
Apparently dynamoDB doesn't let user update the primary key or sort key of an existing item, so if the current itemKey is part of the item's key, we simply move on to the next attribute in the item map. I think this is failing silently because we want to operate on other fields anyway.

some 3 alternative ways that still ensure consistency: 
1) a loop to check the keys (and return err) before the updating loop, 
2) do the same as now, but revert the changes later on if we cannot update the key
3) do the same as now, and return a list of keys that couldn't get updated